src/main/java/pl/futurecollars/invoicing/utils/IdService.java

```diff
@@ -0,0 +1,35 @@
+package pl.futurecollars.invoicing.utils;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import pl.futurecollars.invoicing.db.file.Configuration;
+
+public class IdService {
+
+  private final FilesService filesService;
+  private final Path idPath;
+  private final Path databasePath;
+
+  public IdService(FilesService filesService) {
+    this.filesService = filesService;
+    idPath = Path.of(Configuration.ID_FILE);
```

Comment:
jeśli w jednej linijce konstruktora korzystasz z "this" to w innych również tak rób, będzie spójniej